### PR TITLE
Generate test data for a study

### PR DESCRIPTION
**Pull request by nathanstitt**
Sets up a framework for downloading Qualtrics data

### CODE REVIEW

File: backend/app/controllers/api/v1/researcher/study_responses_controller.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+  def fetch
+    @study.fetch_responses(is_testing: @is_testing)
+    show
+  end
+
+  protected
+
+  def set_study
+    # TODO: set is_testing only when coming from outside network
```
Comment (chrisbendel) [inline backend/app/controllers/api/v1/researcher/study_responses_controller.rb:25]: If we're doing this in the future let's make a ticket so we don't lose sight

File: backend/config/application.rb [hunk 2]
```diff
@@ -21,7 +21,7 @@
 # you've limited to :test, :development, or :production.
 Bundler.require(*Rails.groups)
 
-Dotenv.load('/etc/.env', '.env')
+Dotenv.load('/etc/.env', '.env', '.env.local')
```
Comment (chrisbendel) [inline backend/config/application.rb:24]: Is .env.local something we'll have to set up locally that will be ignored in source control? if so just making sure i don't lose sight of this :D

File: backend/app/models/study_response_export.rb [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+  has_many_attached :files
+
+  def fresh?
+    is_complete && updated_at.after?(4.hours.ago)
+  end
+
+  def fetch
+    return if is_complete?
+    # TODO: kickoff and monitor Qualtrics download
```
Comment (chrisbendel) [inline backend/app/models/study_response_export.rb:22]: lets create a followup ticket before/when we merge

**Review by chrisbendel — APPROVED** [review]

File: backend/app/models/study_response_export.rb (hunk 3 above)
Comment (nathanstitt) [reply backend/app/models/study_response_export.rb:22]: https://github.com/openstax/research/issues/318

**Pull request merged**